src/functionsDelete.js

```diff
@@ -54,7 +54,7 @@ module.exports = function (functionsToDelete, projectId, appEngineLocation) {
   deletes = _.map(functionsToDelete, function (name) {
     const scheduleName = helper.getScheduleName(name, appEngineLocation);
     const topicName = helper.getTopicName(name);
-    const functionName = helper.getFunctionName(name);
+    const functionName = helper.getFunctionId(name);
```

Comment:
Same here - functionName vs Id is getting tricky.

Reply:
+100 on these - i just found another bug caused by this. I was accidentally using name instead of ID for a call to setIamPolicy, and didn't notice until now because that call was failing for me anyway due to the org policy.